lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -37,8 +37,12 @@ use LedgerSMB::Magic qw( EC_EMPLOYEE HTTP_454 PERL_TIME_EPOCH );
 use HTTP::Status qw( HTTP_OK HTTP_UNAUTHORIZED );
 
 my $logger = Log::Log4perl->get_logger('LedgerSMB::Scripts::setup');
-$LedgerSMB::VERSION =~ /(\d+\.\d+)./;
-my $MINOR_VERSION = $1;
+my $MINOR_VERSION;
+if ( $LedgerSMB::VERSION =~ /(\d+\.\d+)./ ) {
+    $MINOR_VERSION = $1;
+} else {
+        die "Error: Invalid data";
```

Comment:
This one is a little trickier.
As long as we continue to use version numbers in the same way it should be fine.
If, for some reason, we change to not using the expected version number format, we shouldn't die.
A warning Message would make sense (after all the change could be a local one for testing purposes)